wagtail/wagtailadmin/templates/wagtailadmin/chooser/browse.html

```diff
@@ -1,19 +1,18 @@
 {% load i18n %}
 {% if page_types_restricted %}
     {% trans "Choose" as choose_str %}
-    {% trans page_type_name as subtitle %}
 {% else %}
     {% trans "Choose a page" as choose_str %}
 {% endif %}
 
-{% include "wagtailadmin/shared/header.html" with title=choose_str subtitle=subtitle search_url="wagtailadmin_choose_page_search" query_parameters="page_type="|add:page_type_string icon="doc-empty-inverse" %}
+{% include "wagtailadmin/shared/header.html" with title=choose_str subtitle=page_type_names|join:", " search_url="wagtailadmin_choose_page_search" query_parameters="page_types="|add:page_type_string icon="doc-empty-inverse" %}
 
 <div class="nice-padding">
     {% include 'wagtailadmin/chooser/_link_types.html' with current='internal' %}
 
     {% if page_types_restricted %}
         <p class="help-block help-warning">
-            {% blocktrans with type=page_type_name %}
+            {% blocktrans with type=page_type_names|join:", " %}
                 Only pages of type "{{ type }}" may be chosen for this field. Search results will exclude pages of other types.
```

Comment:
This message does not read well when there are multiple page types:

> Only pages of type "Content Page, Event Page, Gallery Page" may be chosen for this field...

Consider dropping the quotes, and changing the message depending upon the count using the `{% blocktrans %}{% plural %}` combination